virttest/utils_disk.py

```diff
@@ -259,7 +261,11 @@ def get_windows_disks_index(session, image_size, timeout=60):
         disk_size = str(int(image_size[:-1]) * 1024) + " MB"
     else:
         disk_size = image_size[:-1] + " GB"
-    regex_str = 'Disk (\d+).*?%s.*?%s' % (disk_size, disk_size)
+    if partition:
+        regex_str = 'Disk (\d+).*?%s' % (disk_size)
+    else:
+        regex_str = 'Disk (\d+).*?%s.*?%s' % (disk_size, disk_size)
+
```

Comment:
I think we do not need to add an argument to judge if the disk is made partitioned but handle it by this part.
 Can we set the `regex_str` according to the output of `session.cmd_output(list_disk_cmd, timeout)`?
I mean we can judge whether the disk is new or existing by the output of `list_disk_cmd`.

Reply:
Hi @qingwangrh 
Would you like to provide an example output of `list_disk_cmd` for the new disk and existing disk?
Thanks.

Reply:
DISKPART> list disk

  Disk ###  Status         Size     Free     Dyn  Gpt
  --------  -------------  -------  -------  ---  ---
  Disk 0    Online           32 GB      0 B         
  Disk 1    Online         3072 MB  1070 MB         

--> create a partition on disk1

DISKPART> list disk

  Disk ###  Status         Size     Free     Dyn  Gpt
  --------  -------------  -------  -------  ---  ---
  Disk 0    Online           32 GB      0 B         
  Disk 1    Online         3072 MB  3070 MB  

--> no partition on disk1

Reply:
Do you mean just change regex_str = 'Disk (\d+).*?%s.*?%s' % (disk_size, disk_size) to regex_str = 'Disk (\d+).*?%s' % (disk_size) ?


I think they are different requirements, please refer to def get_linux_disks(session, partition=False)